libs/common/fixtures/src/lib/organisations.fixture.ts

```diff
@@ -1,6 +1,15 @@
 import { deepFreeze } from './utils/freeze'
 import { Organization } from '@geonetwork-ui/common/domain/model/record'
 
+export const REGION_HAUT_DE_FRANCE_ORG_FIXTURE = (): Organization => ({
+  name: 'Région Hauts-de-France',
+  description: 'A very sweet description about Région Hauts-de-France.',
+  email: 'contact@region-haut-de-france.com',
+  website: new URL('https://region-haut-de-france.com/'),
+  logoUrl: new URL('https://region-haut-de-france.com/logo.png'),
+  recordCount: 4,
+})
```

Comment:
I think it would be beneficial to agree on a way to write our fixtures. The original approach was to use `deepFreeze` to avoid mutating fixtures between tests; creating an object each time makes sense too but it'd be better not to mix these practices too much to avoid confusion